packages/theme/src/fontSizes/base.ts

```diff
@@ -3,6 +3,14 @@ import { rem } from "polished";
 import { buildThemeAliases } from "../utils/buildThemeAliases";
 
 const baseFontSizes = {
+  "font-size-700": rem("40px"),
+  "font-size-600": rem("32px"),
+  "font-size-500": rem("28px"),
+  "font-size-400": rem("24px"),
+  "font-size-300": rem("20px"),
+  "font-size-200": rem("16px"),
+  "font-size-100": rem("14px"),
+  "font-size-75": rem("12px"),
```

Comment:
Did you intend to strip out the theme values below?

Reply:
You mean the text aliases? If so, yes since those components have been removed.

Reply:
oh haha, missing context of what's below... I mean did you mean to remove the old base values 8,12,etc. (they are still kicking around)

Reply:
Oooooh, that makes sense. I'll remove em.